utils/appliance/__init__.py

```diff
@@ -163,6 +163,15 @@ def get(self, cls, *args, **kwargs):
         assert 'appliance' not in kwargs
         return cls(appliance=self, *args, **kwargs)
 
+    def postgres_version(self):
+        # postgres's version is in the service name and file paths when we pull it from SCL,
+        # so this is a little resolver to help keep the version picking centralized
+        return version.pick({
+            version.LOWEST: 'postgresql92',
+            '5.5': 'rh-postgresql94',
+            '5.7': 'rh-postgresql95'
+        })
+
```

Comment:
however i do notice, that since we have appliance, and its version, we should use soemthing else to version pick since we dont need to rely on globals there